### ISSUE TITLE
STRETCH - events

### ISSUE THREAD

**Issue by mkcarter7** [mkcarter7/senior-assist#1]
User Story
-family members/caregivers want to use a calendar to display and schedule available times

Acceptance Criteria
-family members/caregivers can see calendar
-family members/caregivers can update calendar
-family members/caregivers can edit calendar

Dependencies
-all MVP must be done before this stretch goal can be done

Dev Notes
-make events
-render
-complete integrations
